src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -920,4 +920,43 @@ public double getAsDouble() {
     public static int safeCastLongToInt(long v) {
         return v > Integer.MAX_VALUE ? Integer.MAX_VALUE : (int) v;
     }
+
+    /**
+     * Scales a proposed recipe voltage according to a provided Material working tier.
+     *
+     * @param voltage     The suggested base voltage for the recipe.
+     * @param workingTier The voltage tier ({@link GTValues#V}) to conform this recipe to.
+     *
+     * @return The new recipe voltage.
+     */
+    public static long scaleVoltage(long voltage, int workingTier) {
+        if (workingTier == GTValues.LV) {
```

Comment:
Use `<=`

Reply:
Don't want to use <=, as then materials at ULV voltage (under half), with a higher tier working tier will not get adjusted to use the VHA value